ydb/core/protos/flat_tx_scheme.proto

```diff
@@ -156,6 +156,25 @@ message TEvLoginResult {
     optional string SanitizedToken = 3;  // sanitized token without signature
 }
 
+message TEvListUsers {
+
+}
+
+message TEvListUsersResult {
+    message TUser {
+        optional string Name = 1;
+        optional bool IsEnabled = 2;
+        optional bool IsLockedOut = 3;
+        optional uint64 CreatedAt = 4;
+        optional uint64 LastSuccessfulAttemptAt = 5;
+        optional uint64 LastFailedAttemptAt = 6;
+        optional uint32 FailedAttemptCount = 7;
+        optional string Password = 8;
```

Comment:
YDB do not store or reveal passwords. And there should be no doubt about that.
I think, that name "password" is misleading here.